easybib/libraries/sns.rb

```diff
@@ -0,0 +1,31 @@
+require 'json'
+
+module EasyBib
+  module SNS
+    def self.sns_notify(node, body)
+      require 'aws-sdk'
+
+      if node.nil?
+        Chef::Log.error 'Missing argument: node (AWS instance name)!'
```

Comment:
Also maybe check for both `node['easybib']['sns']['credentials']` and `node['easybib']['sns']['topic']`not being nil. you can use ruby-fu here: `if node.fetch('easybib', {}).fetch('sns', {})['credentials'].nil?`